src/go/rpk/pkg/cli/cmd/wasm/remove.go

```diff
@@ -0,0 +1,93 @@
+package wasm
+
+import (
+	"github.com/Shopify/sarama"
+	"github.com/spf13/cobra"
+	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/kafka"
+	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/utils"
+)
+
+func NewRemoveCommand(
+	createProduce func(bool, int32) (sarama.SyncProducer, error),
+	adminCreate func() (sarama.ClusterAdmin, error),
+) *cobra.Command {
+
+	command := &cobra.Command{
+		Use:	"remove <name>",
+		Short:	"remove inline WASM function",
+		Args:	cobra.ExactArgs(1),
+		RunE: func(_ *cobra.Command, args []string) error {
+			name := args[0]
+			producer, err := createProduce(false, -1)
+			if err != nil {
+				return err
+			}
+			admin, err := adminCreate()
+			if err != nil {
+				return err
+			}
+			return remove(
+				name,
+				producer,
+				admin,
+			)
+		},
+	}
+
+	return command
+}
+
+/**
+this function create and publish message for removing coprocessor
+message format:
+{
+	key: filename,
+	header: {
+		action: "remove"
+	}
+}
+*/
+func remove(
+	name string, producer sarama.SyncProducer, admin sarama.ClusterAdmin,
+) error {
+	exist, err := kafka.ExistingTopic(admin, kafka.CoprocessorTopic)
+	if err != nil {
+		return err
+	}
+	if exist == false {
+		err := kafka.CreateCoprocessorTopic(admin)
+		return err
+	}
+	// create empty message, the remove command doesn't need
+	// information on message, just a key value
+	var emptyMessage []byte
+	header, err := createActionHeader("remove")
+	if err != nil {
+		return err
+	}
+	//publish message
+	return kafka.PublishMessage(
+		emptyMessage,
+		name,
+		producer,
+		kafka.CoprocessorTopic,
+		[]sarama.RecordHeader{header},
+	)
+}
+
+func createActionHeader(action string) (sarama.RecordHeader, error) {
+	// create key
+	key, err := utils.WriteString([]byte{}, "action")
```

Comment:
If you need a `[]byte` from a `string`, it's easier to do `[]byte(str)`. Unless I'm missing something that's happening behind the scenes.

Reply:
Yes, the result `[]bytes` has an RPC format, in this case, `NumberBytes+Bytes`